components/strava/strava.app.js

```diff
@@ -1,21 +1,29 @@
 // Strava API app file
 const axios = require("axios");
```

Comment:
It would be great if you could remove this usage of `axios` and use only `@pipedream/platform`

Reply:
done